lib/GADS/Column.pm

```diff
@@ -1546,28 +1546,40 @@ sub validate_search_date
 # possible (e.g. integer)
 sub resultset_for_values {};
 
-sub values_beginning_with
-{   my ($self, $match_string, %options) = @_;
+sub match_result {
```

Comment:
This is a sub encapsulating the common part of `values_beginning_with` between Column and Person